From 31f023e3f7e8f877b912fad7318cb8ecc920a789 Mon Sep 17 00:00:00 2001
From: Nitin Rawat <nitin.rawat@intel.com>
Date: Tue, 25 Jun 2019 12:44:59 +0530
Subject: [PATCH] [REVERT]Disable nl80211 register for Auth Frame

If AP registers for auth frame,HotSpot and Wifi Direct
is not working.

Fix the issue by disabling the registration
of Auth Frame from wpa_supplicant.

Change-Id: I06e87cac4477f3c555c55cdca80d1f873d9ca03c
Tracked-On:
Signed-off-by: Nitin Rawat <nitin.rawat@intel.com>
---
 src/drivers/driver_nl80211.c | 10 ----------
 1 file changed, 10 deletions(-)

diff --git a/src/drivers/driver_nl80211.c b/src/drivers/driver_nl80211.c
index 21d1398e..8c44dd2f 100644
--- a/src/drivers/driver_nl80211.c
+++ b/src/drivers/driver_nl80211.c
@@ -2470,16 +2470,6 @@ static int nl80211_mgmt_subscribe_ap_dev_sme(struct i802_bss *bss)
 	if (nl80211_action_subscribe_ap(bss))
 		goto out_err;
 
-	if (bss->drv->device_ap_sme) {
-		u16 type = (WLAN_FC_TYPE_MGMT << 2) | (WLAN_FC_STYPE_AUTH << 4);
-
-		/* Register for all Authentication frames */
-		if (nl80211_register_frame(bss, bss->nl_mgmt, type, NULL, 0)
-		    < 0)
-			wpa_printf(MSG_DEBUG,
-				   "nl80211: Failed to subscribe to handle Authentication frames - SAE offload may not work");
-	}
-
 	nl80211_mgmt_handle_register_eloop(bss);
 	return 0;
 
-- 
2.21.0

